MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/location/LocationComponent.java

```diff
@@ -610,6 +610,9 @@ public void setCameraMode(@CameraMode.Mode int cameraMode,
                             @Nullable Double zoom, @Nullable Double bearing, @Nullable Double tilt,
                             @Nullable OnLocationCameraTransitionListener transitionListener) {
     checkActivationState();
+    if (!isEnabled) {
+      return;
+    }
```

Comment:
I think this can break some of the existing implementations. If someone was setting the camera mode before enabling the component, that logic will not work anymore. Is there another way that we can resolve the issue?

Reply:
Then we might need to save the enable state in `LocationCameraController` and only save its camera mode without update camera when disabled.